ngraph_bridge/ie_executable.h

```diff
@@ -42,6 +42,7 @@ class IE_Executable final : public Executable {
  private:
   InferenceEngine::CNNNetwork m_network;
   InferenceEngine::InferRequest m_infer_req;
+  unordered_map<string, shared_ptr<ngraph::runtime::Tensor>> m_params;
```

Comment:
consider changing name to m_fake_params

Reply:
Thanks. Changed to `m_hoisted_params` which I think is more descriptive, and added a small note.